locksmith/config/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "development": {
+    "username": "locksmith_development",
```

Comment:
Probably not for this PR but we should probably consider using env variables here, right?

Reply:
And docker secrets in ci